dfinity/nns-dapp
# NNS1-2851: Remove SnsTransactionModal

# Motivation

Unify wallet implementation.

In this PR, remove SnsTransactoinModal and use IcrcTokenTransactionModal instead.

# Changes

* Add parameters `universeId` to `IcrcTokenTransactionModal` to be used for the TransactionModal and `ledgerCanisterId` is used for the action.
* Pass the `universeId` parameter in the usages of `IcrcTokenTransactionModal`.
* Use `IcrcTokenTransactionModal` in the tokens +page.svelte.
* Remove SnsTransactionModal.

# Tests

* Remove SnsTransactionModal spec file and page object.
* Fix api expect in tokens page spec file.
* Use the icrc transaction modal in the `TokensRoutePo` for sns transactions.

# Todos


Not necessary.
